lib/ide/parser/src/wsclient.rs

```diff
@@ -84,7 +85,7 @@ impl From<serde_json::error::Error> for Error {
 /// All request supported by the Parser Service.
 #[derive(Debug, serde::Serialize, serde::Deserialize)]
 pub enum Request {
-    ParseRequest { program: String },
+    ParseRequest {program: String, ids: IDMap },
```

Comment:
Have I missed something or is the Scala parser-service still needing the same change?